src/main/kotlin/me/owdding/lib/overlays/Overlay.kt

```diff
@@ -0,0 +1,69 @@
+package me.owdding.lib.overlays
+
+import earth.terrarium.olympus.client.ui.context.ContextMenu
+import me.owdding.lib.mixins.OverlayAccessor
+import net.minecraft.client.gui.GuiGraphics
+import net.minecraft.network.chat.Component
+import tech.thatgravyboat.skyblockapi.helpers.McClient
+import tech.thatgravyboat.skyblockapi.helpers.McScreen
+import tech.thatgravyboat.skyblockapi.utils.text.Text
+
+interface Overlay {
+
+    val modName: Component
```

Comment:
I'd prefer if we don't do this and make it so you have to provide the mod id and create it with that because it'd make it far easier for getting 1 mods specific overlays